client/src/pages/RentCheckout.jsx

```diff
@@ -0,0 +1,239 @@
+import { useState } from 'react'
+import { ChevronRightIcon } from '@heroicons/react/solid'
+import { RadioGroup } from '@headlessui/react'
+
+const steps = [
+    { name: 'Select NFT', href: '/rent', status: 'complete' },
+    { name: 'Rental Information', href: '#', status: 'current' },
+    { name: 'Swap Tokens', href: '#', status: 'upcoming' },
+    { name: 'Confirmation', href: '#', status: 'upcoming' },
+]
+const nft = {
+    id: 1,
+    name: 'Bored Ape Yacht Club',
+    price: '50',
+    token: '#3',
+    imageSrc:
+        'https://ipfs.io/ipfs/QmYxT4LnK8sqLupjbS6eRvu1si7Ly2wFQAqFebxhWntcf6',
+    imageAlt: 'Bored Ape Yacht Club #3',
+}
+const days = [
+    { name: '1', available: true },
+    { name: '2', available: true },
+    { name: '3', available: true },
+    { name: '5', available: true },
+    { name: '7', available: true },
+    { name: '14', available: true },
+    { name: '21', available: true },
+    { name: '28', available: true },
+]
+
+function classNames(...classes) {
```

Comment:
I installed the `classnames` package if you want to use that.

Reply:
I can refactor that in my next PR